docs/guides/usage/tracing.livemd

```diff
@@ -190,6 +234,7 @@ For example, most `Integer` functions take an optional `base` argument, defaulti
 If we are only interested in cases where we are parsing a string into an integer with the non-default decimal base, we can trace calls to `Integer.parse` with an arity of `2`—that is, functions were we are specifying an extra `base` argument:
 
 ```elixir
+# NB we're invoking calls function here.
```

Comment:
What does NB mean in this context? Can we expand the acronym to avoid assumptions about the reader's familiarity with slang?

Reply:
I fixed this one Chris.